app/views/post/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="posts-new-comment">
+  <h1>New Post</h1>
+  <div class="posts-new-form">
+    <%= form_with model: @post, url: user_posts_path, scope: :post do |form| %>
+      <div class="text-area">
+        <%= form.text_field :title, placeholder: 'Post Title', required: true, class:"text-form" %>
+      </div>
+      <div>
+        <%= form.text_area :text, placeholder: 'Enter your post here...', rows:14, cols:40, required: true, class:"text-area" %>
+      </div>
+      <div class="create-post-button button-class2">
+        <%= form.submit "Create Post" %>
+      </div> 
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ ] You forgot to put _**s**_ at the end of the `post` folder. The path should be `app/views/posts/new.html.erb` but what you have is `app/views/post/new.html.erb`. Emphasis on the _**s**_ for post. This is necessary because Ruby on Rails uses naming conventions to link controller, route, model and view (MVC). 👍 

![image](https://user-images.githubusercontent.com/77106962/163676838-15b80c1a-0154-4ac9-a57d-b920c8f53926.png)

####  After renaming to `posts`, in your migrations, kindly add `author_id` as the foreign key to comments, posts and likes. 
` t.references :author, foreign_key: { to_table: 'users'}, null: false`

If you don't do it, you will get an error whenever you try to create a post, comment or likes. Remember you referenced it in your model `user.rb`:

```
class User < ApplicationRecord
  has_many :posts, foreign_key: :author_id
  has_many :comments, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id

  def load_last_three_posts
    Post.last(3)
  end
end
```